tools/create-translation-issues.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/bash
+# This script opens translation issues for all languages via the Github API.
+# The script is idempotent and can be run multiple times. It will not create
+# multiple issues if milestone and title remain the same. Instead, it will
+# update any existing issues with a (potentially) updated body text.
+#
+# Syntax:
+#   ./tools/create-translation-issues.sh RELEASE DEADLINE app|web [EXTRA_TEXT]
+#
+# Requirements:
+# - Github CLI tools + valid login
+# - jamulus and jamulussoftware git checkouts
+#
+# Example usage for app translations:
+#   ./tools/create-translation-issues.sh 3.8.0 2021-05-15 app 'Note: The term "Central server" has been replaced with "Directory server"'
+# Example usage for web translations:
+#  ./jamulus/tools/create-translation-issues.sh 3.8.0 2021-05-15 web 'Note: The term "Central server" has been replaced with "Directory server"'
+
+set -eu
+
+if [[ -z ${1:-} ]] || [[ -z ${2:-} ]] || [[ -z ${3:-} ]] ; then
+    echo "Syntax: $0 RELEASE DEADLINE app|web [EXTRA_TEXT]"
+    exit 1
+fi
+
+RELEASE=$1
+DEADLINE=$2
+TYPE=$3
+EXTRA_TEXT=${4:-}
+MILESTONE="Release ${RELEASE}"
+PROJECT=Tracking
+
+declare -A TRANSLATORS_BY_LANG=(
+    # Syntax:
+    # [TYPE_LANG]="github-handle1,github-handle2"
+    # with TYPE being either app or web and
+    # with LANG being the language code (different syntax for app and web!)
+
+    # App translators:
+    [app_de_DE]="rolamos"
+    [app_es_ES]="ignotus666"
+    [app_fr_FR]="jujudusud"
+    [app_it_IT]="dzpex"
+    [app_nl_NL]="henkdegroot,jerogee"
+    [app_pl_PL]="SeeLook"
+    [app_pt_BR]="melcon"
+    [app_pt_PT]="Snayler"
+    [app_sk_SK]="jose1711"
+    [app_sv_SE]="genesisproject2020"
+
+    # Web translators:
+    [web_de]="Helondeth,ewarning"
+    [web_es]="ignotus666"
+    [web_fr]="jujudusud,trebmuh"
+    [web_it]="dzpex"
+)
+
+BODY_TEMPLATE_APP='Hi ${SPLIT_TRANSLATORS},
+We are getting ready for the ${RELEASE} release. No further changes to translatable strings are currently planned for this release.
+
+We would be happy if you updated the Jamulus software translations for **${LANG}** until **${DEADLINE}**.
+
+Please
+- Update your fork to current `master`,
```

Comment:
`- Update your fork to current ``master``,`
This should say `from jamulussoftware/jamulus master` or `from jamulussoftware/jamuluswebsite master` (depending...)

Reply:
Updated.